### PR TITLE
removing Nippy from equation

### PR DESCRIPTION
**Pull request by elisehuard**

### CODE REVIEW

File: src/kixi/comms/messages.clj [hunk 1]
```diff
@@ -95,3 +96,16 @@
     (if command-id
       (assoc r :kixi.comms.command/id command-id)
       r)))
+
+(defn edn-to-bytebuffer
+  [edn]
+  (let [data (str edn)
+        data-in-bytes (byte-array (map (comp byte int) data))
```
Comment (acron0) [inline src/kixi/comms/messages.clj]: `data-in-bytes` looks unused here

File: src/kixi/comms/messages.clj [hunk 2]
```diff
@@ -95,3 +96,16 @@
...
+        buf (ByteBuffer/wrap (.getBytes data))]
+    buf)
+  )
+
+(defn bytebuffer-to-edn [byte-buffer]
+  (let [b (byte-array (.remaining byte-buffer))]
+    (.get byte-buffer b)
```
Comment (acron0) [inline src/kixi/comms/messages.clj]: This sexp looks spurious - is it side-effecting?

File: src/kixi/comms/messages.clj (hunk 1 above)
Comment (elisehuard) [inline src/kixi/comms/messages.clj]: good point, was one of my experiments

File: src/kixi/comms/messages.clj (hunk 2 above)
Comment (elisehuard) [inline src/kixi/comms/messages.clj]: yes, indeed, both do.  Bytebuffers are icky.

**Pull request merged**